Frends.AzureBlobStorage.DownloadBlob/Frends.AzureBlobStorage.DownloadBlob/Definitions/Options.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+#pragma warning disable CS1591

```

Comment:
Why is there a pragma here? Is it needed?

Reply:
Removed.